murdock/murdock.py

```diff
@@ -122,18 +141,37 @@ async def shutdown(self):
                 LOGGER.debug(f"Stopping {job}")
                 await job.stop()
 
+    def _set_worker_metric(self, job: MurdockJob, state: str):
+        current_task = asyncio.current_task().get_name()
```

Comment:
This makes the assumption that there's only one worker at a time and Murdock can be configured to have several.

Reply:
I thought this would get me a unique identifier for a worker. What's the correct way to get the unique identifier of a murdock worker?

Reply:
Preferably an identifier that is human readable and thus can be used as label value for the metrics

Reply:
The current code behaves as I would expect with the `test_schedule_multiple_jobs` test function, it exposes two samples, one for each worker with the following labels
```
{'worker': 'MurdockWorker_0', 'uuid': 'f6050c87babf45e1ba2e353b37c61162', 'commit': 'test_commit', 'title': 'PR #1 (test_co)', 'creation_time': 1669381054645, 'fasttracked': False, 'state': 'running'}
```
And
```
{'worker': 'MurdockWorker_1', 'uuid': '1a10ef7cfaa14eeda8e1eec3ea956e0f', 'commit': 'test_commit', 'title': 'PR #2 (test_co)', 'creation_time': 1669381054645, 'fasttracked': False, 'state': 'running'}
```

Or is there another case I'm not catching here?